<launch>
  <!-- Start the SR4500 -->
  <include file="$(find swissranger_camera)/launch/sr_eth.launch">
    <arg name="device_ip" value="192.168.1.31" />
  </include>

   <!-- Launch ground based people detection node -->
   <node pkg="detection" type="ground_based_people_detector_sr" name="ground_based_people_detector_sr" output="screen" respawn="true">
    <rosparam command="load" file="$(find detection)/conf/ground_based_people_detector_sr.yaml" /> 
    <param name="classifier_file" value="$(find detection)/data/HogSvmPCL.yaml"/>
    <param name="pointcloud_topic" value="/SwissRanger/pointcloud2_raw"/>
    <param name="confidence_topic" value="/SwissRanger/confidence/image_raw"/>
    <param name="camera_info_topic" value="/SwissRanger/camera_info"/>
    <param name="output_topic" value="/detector/detections"/>
    <param name="rate" value="60.0"/>  
  </node>

</launch>
